docs/articles/nunit-engine/extensions/Project-Loaders.md

```diff
@@ -4,7 +4,7 @@ uid: ProjectLoaders
 
 # Project Loaders
 
-**Project Loaders** are extensions that know how to load a project in a particular format and create a package suitable for running tests under NUnit. NUnit itself provides two of them:
+**Project Loaders** are extensions that know how to load a project in a particular format and create a package suitable for running tests under NUnit. The NUnit Organization itself provides two of them:
```

Comment:
"The NUnit Project" is what we decided to call ourselves after lots of hours of discussion.